bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix: deploy/service and deploy/token-refresher make target errors

## Description
Fixes the following issues with some deploy targets:
- Fix error "Unknown flag -P" when make deploy/token-refresher is ran. 
- Ensure that resources for deploy/service is actually applied to the cluster.
- Ensure that the `OBSERVABILITY_CONFIG_TAG` param is actually applied
- Update `OBSERVABILITY_CONFIG_TAG` default value to the latest observability-mk-config repo release

## Verification Steps
1. Run make deploy/token-refresher (ensure it deploys successfully)
2. Run make deploy/service (ensure resources are created on cluster)

## Checklist (Definition of Done)
